nncf/api/compression.py

```diff
@@ -321,6 +321,15 @@ def get_transformation_layout(self, model: ModelType) -> TransformationLayout:
             a list of algorithm-specific modifications.
         """
 
+    @abstractmethod
+    def initialize(self, model: ModelType) -> None:
```

Comment:
If this is only going to be called internally, this should be a protected method.

Reply:
Guys, I agree with your point, but this method is used not only internally. I mean, the composite compression builder calls this method both in the TF and PT backends https://github.com/openvinotoolkit/nncf/blob/480f4a57e7bfac7ddfea0e0379d571ab10877939/beta/nncf/tensorflow/api/composite_compression.py#L56, https://github.com/openvinotoolkit/nncf/blob/480f4a57e7bfac7ddfea0e0379d571ab10877939/nncf/torch/composite_compression.py#L101

I guess, that the same issue we have with `get_transformation_layout`.